src/App/App.css

```diff
@@ -0,0 +1,151 @@
+/* * {
+  border: rgb(141, 141, 242) solid 1.5px;
+  background-color: #e9dfcc;
+} */
+
+.App {
+  margin-left: auto;
+  margin-right: auto;
+  min-width: 320px;
```

Comment:
You could use a px to em converter to make all of your styling responsive: https://www.w3schools.com/tags/ref_pxtoemconversion.asp

Reply:
This particular min-width is in pixels to limit the minimum amount of width using the smallest size phone screen as a hard stop.  Thanks for that other resource though.  Will look into it for general use.